sop-core/src/Data/SOP/Sing.hs

```diff
@@ -93,7 +98,7 @@ deriving instance Eq   (Shape xs)
 deriving instance Ord  (Shape xs)
 
 -- | The shape of a type-level list.
-shape :: forall (xs :: [k]). SListI xs => Shape xs
+shape :: forall k (xs :: [k]). SListI xs => Shape xs
```

Comment:
does this change `TypeApplications` behaviour?

Reply:
No. `k` was always available for `TypeApplications` even before this patch, since it's mentioned explicitly in the type signature. This patch does not change anything in that regard.

Reply:
The use of `shape` is still possible without `TypeInType` for old GHCs, right?

Reply:
I assume you're thinking of call sites for `shape`? They shouldn't need to enable `TypeInType`, no.